src/Php73/bootstrap.php

```diff
@@ -27,5 +27,5 @@ function hrtime($asNum = false) { return p\Php73::hrtime($asNum); }
     function array_key_first(array $array) { foreach ($array as $key => $value) { return $key; } }
 }
 if (!function_exists('array_key_last')) {
-    function array_key_last(array $array) { end($array); return key($array); }
+    function array_key_last(array $array) { key(array_slice($array, -1, 1, true)); }
```

Comment:
funny that we missed the return statement :)

Reply:
fixed in df3c863

Reply:
isn't the testsuite catching this kind of mistakes ?

Reply:
yes it is, that's how I spotted it

Reply:
then we have an issue in our process if we merge PRs when the CI tells us it's broken...

Reply:
Sorry about that!

CI were broken at that time https://travis-ci.org/github/symfony/polyfill/jobs/737662800 

Did 2 mistake:
I didn't ran test locally (I often count on CI for that)
I didn't marked the PR as draft

truly sorry